5.3Parseltongue.py

```diff
@@ -0,0 +1,20 @@
+def find_secrets(file_name):
+    """
+    find the secret messages in the resources/logo.jpg file
+    :param file_name: the resources/logo.jpg file
+    :return: generator for the secret messages
+    """
+    with open(file_name, 'rb') as file:
+        word = b''
+        while True:
```

Comment:
Don't use while true. Plan the code so you can use the break condition as the while condition instead.